commit ed44c187d5628339adea6a59124ceef1636085b1
Author:     Tizen Build Automation <tizenbuild@samsung.com>
AuthorDate: Fri Oct 23 01:41:23 2026 +0000
Commit:     Tizen Build Automation <tizenbuild@samsung.com>
CommitDate: Fri Oct 23 01:41:23 2026 +0000

    Update candidate_snapshot : TIZEN/Tizen/Tizen-Base-X-ASAN/project_meta,TIZEN/Tizen/Tizen-Unified-SC/project_meta,TIZEN/Tizen/Tizen-Base-X/project_meta,TIZEN/Tizen/Tizen-Base-Dev/project_meta,TIZEN/Tizen/Tizen-Unified-Dev/project_meta,TIZEN/Tizen/Tizen-Unified/project_meta,TIZEN/Tizen/Tizen-Base-SC/project_meta,TIZEN/Tizen/Tizen-Base/project_meta,TIZEN/Tizen/Tizen-Rust/project_meta,TIZEN/Tizen-9.0/Tizen-9.0-Base/project_meta,TIZEN/Tizen-10.1/Tizen-10.1-Base/project_meta,TIZEN/Tizen-10.1/Tizen-10.1-Unified/project_meta,
---
 TIZEN/Tizen-10.1/Tizen-10.1-Base/candidate_snapshot    | 2 --
 TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot | 2 --
 TIZEN/Tizen-9.0/Tizen-9.0-Base/candidate_snapshot      | 1 -
 TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot          | 2 --
 TIZEN/Tizen/Tizen-Base-SC/candidate_snapshot           | 1 -
 TIZEN/Tizen/Tizen-Base-X-ASAN/candidate_snapshot       | 1 -
 TIZEN/Tizen/Tizen-Base-X/candidate_snapshot            | 1 -
 TIZEN/Tizen/Tizen-Base/candidate_snapshot              | 1 -
 TIZEN/Tizen/Tizen-Rust/candidate_snapshot              | 1 -
 TIZEN/Tizen/Tizen-Unified-Dev/candidate_snapshot       | 1 -
 TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot        | 1 -
 TIZEN/Tizen/Tizen-Unified/candidate_snapshot           | 3 ---
 12 files changed, 17 deletions(-)

diff --git a/TIZEN/Tizen-10.1/Tizen-10.1-Base/candidate_snapshot b/TIZEN/Tizen-10.1/Tizen-10.1-Base/candidate_snapshot
index 5d86753..b9807a6 100644
--- a/TIZEN/Tizen-10.1/Tizen-10.1-Base/candidate_snapshot
+++ b/TIZEN/Tizen-10.1/Tizen-10.1-Base/candidate_snapshot
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
-  <candidate target="tizen-10.1-base_20260922.223047" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
   <candidate target="tizen-10.1-base_20261019.081329" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
   <candidate target="tizen-10.1-base_20261022.033837" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
-  <base_candidate target="tizen-10.1-base_20260922.223047" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
   <base_candidate target="tizen-10.1-base_20261019.081329" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
   <base_candidate target="tizen-10.1-base_20261022.033837" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot b/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
index f832b2d..b4c3a56 100644
--- a/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
+++ b/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
-  <candidate target="tizen-10.1-unified_20261012.080519" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261013.101057" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261014.010544" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261014.210343" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
@@ -16,7 +15,6 @@
   <candidate target="tizen-10.1-unified_20261021.210844" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261022.043909" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
-  <base_candidate target="tizen-10.1-base_20260922.223047" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
   <base_candidate target="tizen-10.1-base_20261019.081329" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
   <base_candidate target="tizen-10.1-base_20261022.033837" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Base/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen-9.0/Tizen-9.0-Base/candidate_snapshot b/TIZEN/Tizen-9.0/Tizen-9.0-Base/candidate_snapshot
index 64b4910..d7ef3c1 100644
--- a/TIZEN/Tizen-9.0/Tizen-9.0-Base/candidate_snapshot
+++ b/TIZEN/Tizen-9.0/Tizen-9.0-Base/candidate_snapshot
@@ -5,7 +5,6 @@
   <candidate target="tizen-9.0-base_20261019.081230" url="http://download.tizen.org/snapshots/TIZEN/Tizen-9.0/Tizen-9.0-Base/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <base_candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
diff --git a/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot
index bf62244..70abe4a 100644
--- a/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot
@@ -2,10 +2,8 @@
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
   <candidate target="tizen-base-dev_20260930.023443" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <candidate target="tizen-base-dev_20261012.145954" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
   <candidate target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
   <base_candidate target="tizen-base-dev_20260930.023443" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <base_candidate target="tizen-base-dev_20261012.145954" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
   <base_candidate target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Base-SC/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-SC/candidate_snapshot
index 3fd995b..4e84fef 100755
--- a/TIZEN/Tizen/Tizen-Base-SC/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-SC/candidate_snapshot
@@ -5,7 +5,6 @@
   <candidate target="tizen-base-sc_20261007.051001" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-SC/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <base_candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
diff --git a/TIZEN/Tizen/Tizen-Base-X-ASAN/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-X-ASAN/candidate_snapshot
index 2c80ff2..0dde773 100644
--- a/TIZEN/Tizen/Tizen-Base-X-ASAN/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-X-ASAN/candidate_snapshot
@@ -5,7 +5,6 @@
   <candidate target="tizen-base-x-asan_20250924.111414" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-X-ASAN/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <base_candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
diff --git a/TIZEN/Tizen/Tizen-Base-X/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-X/candidate_snapshot
index 5101d0d..7fd5cf4 100644
--- a/TIZEN/Tizen/Tizen-Base-X/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-X/candidate_snapshot
@@ -5,7 +5,6 @@
   <candidate target="tizen-base-x_20261020.230437" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-X/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <base_candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
diff --git a/TIZEN/Tizen/Tizen-Base/candidate_snapshot b/TIZEN/Tizen/Tizen-Base/candidate_snapshot
index 907f13c..c0a7500 100644
--- a/TIZEN/Tizen/Tizen-Base/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base/candidate_snapshot
@@ -2,7 +2,6 @@
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
diff --git a/TIZEN/Tizen/Tizen-Rust/candidate_snapshot b/TIZEN/Tizen/Tizen-Rust/candidate_snapshot
index b84b4c8..5703703 100644
--- a/TIZEN/Tizen/Tizen-Rust/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Rust/candidate_snapshot
@@ -5,7 +5,6 @@
   <candidate target="tizen-rust_20260513.070440" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Rust/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <base_candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
diff --git a/TIZEN/Tizen/Tizen-Unified-Dev/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified-Dev/candidate_snapshot
index e847b8e..f83e7f1 100644
--- a/TIZEN/Tizen/Tizen-Unified-Dev/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified-Dev/candidate_snapshot
@@ -4,6 +4,5 @@
   <candidate target="tizen-unified-dev_20261015.090738" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-Dev/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
   <base_candidate target="tizen-base-dev_20260930.023443" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <base_candidate target="tizen-base-dev_20261012.145954" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
   <base_candidate target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot
index 70f0575..f4b7ae3 100755
--- a/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified-SC/candidate_snapshot
@@ -7,7 +7,6 @@
   <candidate target="tizen-unified-sc_20261021.232527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-SC/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <base_candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
diff --git a/TIZEN/Tizen/Tizen-Unified/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
index 0de885c..cd0224d 100644
--- a/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
-  <candidate target="tizen-unified_20261012.092209" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
-  <candidate target="tizen-unified_20261012.235139" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261013.105300" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261013.211222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261014.063636" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
@@ -17,7 +15,6 @@
   <candidate target="tizen-unified_20261022.090156" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
-  <base_candidate target="tizen-base_20260930.045222" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
   <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
